app/views/keywords/new.html.slim

```diff
@@ -0,0 +1,7 @@
+= form_with(model: keyword, local: true, url: keywords_path) do |form|
+  h5.mb-3 Upload keywords CSV file
```

Comment:
i18n

Reply:
My bad. Fixed in ff12ff9051a53fb96c1ac1ccdc5d97963aa1d3b8